src/plugins/controls/public/controls/data_controls/options_list_control/components/options_list_control.tsx

```diff
@@ -121,18 +121,15 @@ export const OptionsListControl = ({
                 )}
                 delay="long"
               >
-                <EuiToken
+                <EuiIcon
                   tabIndex={0}
-                  iconType="alert"
+                  type="warningFilled"
                   size="s"
-                  color="euiColorVis5"
-                  shape="square"
-                  fill="dark"
+                  color="warning"
                   title={OptionsListStrings.control.getInvalidSelectionWarningLabel(
                     invalidSelections.size
                   )}
                   data-test-subj={`optionsList__invalidSelectionsToken-${api.uuid}`}
-                  css={{ verticalAlign: 'text-bottom' }} // Align with the notification badge
```

Comment:
This doesn't look right, either:

| Main (Amsterdam) | This PR (Borealis) |
|--------|--------|
| ![image](https://github.com/user-attachments/assets/62ff5605-1b3d-4d55-918b-51fd4949e3c1) | ![image](https://github.com/user-attachments/assets/4891bfcb-63c4-4089-8633-fcc12afbbef2) |

Reply:
Like I said above, I'll break out this warning icon change into a separate PR so we can polish how it looks. It looks like that `textWarning` color is too dark, but maybe we can use a different warning color.